remotes/docker/fetcher.go

```diff
@@ -60,7 +60,7 @@ func (r dockerFetcher) Fetch(ctx context.Context, desc ocispec.Descriptor) (io.R
 				log.G(ctx).WithError(err).Debug("failed to parse")
 				continue
 			}
-			log.G(ctx).Debug("trying alternative url")
+			log.G(ctx).Info("request")
```

Comment:
maybe add more information here?

Reply:
I'm not sure. It may result in leaking some secret stuff.

Reply:
It should at least be clear the request is from `dest.URLs`

Reply:
I think it should contain the URL....there is not too much information in ctx...

Reply:
Because of `log.WithLogger(ctx, log.G(ctx).WithField("url", us))`, log.G returns a logger with url=us. Let me double-check.